packages/zosfiles/__tests__/__integration__/cli/create/zfs/__snapshots__/cli.files.create.zfs.integration.test.ts.snap

```diff
@@ -0,0 +1,161 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Create z/OS File System should display the help 1`] = `
+"================Z/OS FILES CREATE ZOS-FILE-SYSTEM HELP===============
+
+ COMMAND NAME
+ ------------
+
+   zos-file-system | zfs
+
+ DESCRIPTION
+ -----------
+
+   Create a z/OS file system
+
+ USAGE
+ -----
+
+   zowe zos-files create zos-file-system <fileSystemName> [options]
+
+ POSITIONAL ARGUMENTS
+ --------------------
+
+   fileSystemName		 (string)
+
+      The name of the file system to mount
+
+ OPTIONS
+ -------
+
+   --cyls-pri  | --cp (number)
+
+      The number of primary cylinders to allocate for the ZFS
+
+      Default value: 10
+
+   --cyls-sec  | --cs (number)
+
+      The number of secondary cylinders to allocate for the ZFS
+
+      Default value: 2
+
+   --data-class  | --dc (string)
+
+      The SMS data class to use for the allocation
+
+   --group  | -g (string)
+
+      The z/OS group ID or GID for the group of the ZFS root directory
+
+   --management-class  | --mc (string)
+
+      The SMS management class to use for the allocation
+
+   --owner  | -o (string)
+
+      The z/OS user ID or UID for the owner of the ZFS root directory
+
+   --perms  | -p (number)
+
+      The permissions code for the ZFS root directory
+
+      Default value: 755
+
+   --storage-class  | --sc (string)
+
+      The SMS storage class to use for the allocation
+
+   --timeout  | -t (number)
+
+      The number of seconds to wait for the create command to complete
+
+      Default value: 20
+
+   --volumes  | -v (array)
+
+      The storage volumes on which to allocate the z/OS file system. Specify a single
+      volume by its volume serial (VOLSER). To specify more than one volume, enclose
```

Comment:
In bash, double quotes for an array option aren't required and different shells/terminals may have different requirements.